cmd/scollector/collectors/apache_mod_info.go

```diff
@@ -0,0 +1,112 @@
+package collectors
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"golang.org/x/net/html"
```

Comment:
The adds a new third-party dependency. Since its code could change at any time and thus break scollector, we use a tool that copies over the needed files and changes import paths to our copy. To install and run that tool:

```
go get github.com/mjibson/party
cd $GOPATH/src/bosun.org
party
```